webpack.config.js

```diff
@@ -1,16 +1,9 @@
 var path = require('path')
 
-var pkg = require('./package.json')
-var tags = makeTags(pkg.version)
-
-var entry = {}
-entry['cf-extension-api'] = './lib/api/index.js'
-tags.forEach((tag) => {
-  entry[`tagged/cf-extension-api-v${tag}`] = './lib/api/index.js'
-})
-
 module.exports = {
-  entry: entry,
+  entry: {
+    'cf-extension-api': './lib/api/index.js'
```

Comment:
This a lot better already, but Is there a reason that we use an object as an entry here since we have on single entry why not having `entry:'./lib/api/index.js'`